study.md

```diff
@@ -50,7 +50,25 @@ A kitchen has many items.
     rack, and cinnamon, nutmeg, and vanilla on another shelf.
 
 ```json
-<!-- your answer here -->
+<!-- kitchen
+      cupboard
+          bowls
+          plates
+      stove-electric
+        heating elements
+          small
+          medium
+          large
+          extra large
+      rack
+        herbs
+          chives
+          lemongrass
+          mint
+        spices
+          cinnamon
+          nutmeg
+          vanilla-->
```

Comment:
Hey Lisa we wanted you to structure it like JSON which is basically just like a JavaScript object. Something more like this:

```
{ kitchen: {
    cupboard: [
      'bowls',
      'plates'
    ]
  }
}
```